src/calendar/internal.tsx

```diff
@@ -33,6 +33,7 @@ export default function Calendar({
   onChange,
   __internalRootRef,
   i18nStrings,
+  granularity = 'day',
```

Comment:
Move this up to the public interface in `index.tsx` so the documenter can pick it up correctly.

Reply:
Thanks, updated.